addon/components/listbox/-button.hbs

```diff
@@ -0,0 +1,18 @@
+{{#let (element (or @as 'button')) as |Tag|}}
```

Comment:
The other components so far have used `@tagName` to set an alternate element that can be used as root element; I agree that `@as` is a better name, but for now it makes more sense to be consistent with the usage of `@tagName` and migrate them _all_ to use `@as` down the road